### PR TITLE
[demo/cartservice] Add support for logs

### PR DESCRIPTION
**Pull request by Kielek**
Needed for https://github.com/open-telemetry/opentelemetry-demo/pull/1086

### CODE REVIEW

File: content/en/docs/demo/services/cart.md [hunk 1]
```diff
@@ -9,8 +9,9 @@ with a Redis caching service for fast access to shopping cart data.
 
 [Cart service source](https://github.com/open-telemetry/opentelemetry-demo/blob/main/src/cartservice/)
 
-> **Note** OpenTelemetry for .NET uses the `System.Diagnostic` library as its
-> API in lieu of the standard OpenTelemetry API.
+> **Note** OpenTelemetry for .NET uses the `System.Diagnostic.DiagnosticSource`
+> library as its API in lieu of the standard OpenTelemetry API for Traces and
```
Comment (theletterf) [inline content/en/docs/demo/services/cart.md:13]:
```suggestion
> library as its API instead of the standard OpenTelemetry API for Traces and
```

**Review by theletterf — APPROVED** [review]

**Review by lachmatt — APPROVED** [review]

**Review by julianocosta89 — APPROVED** [review]
Thank you for updating it @Kielek !

File: content/en/docs/demo/services/cart.md [hunk 2]
```diff
@@ -91,4 +92,11 @@ builder.Services.AddOpenTelemetry()
 
 ## Logs
 
-TBD
+Logs should be configured in .NET DI container on `LoggingBuilder` level by
```
Comment (svrnm) [inline content/en/docs/demo/services/cart.md:95]:
```suggestion
Logs should be configured in .NET dependency injection container on `LoggingBuilder` level by
```

Comment (Kielek) [reply content/en/docs/demo/services/cart.md]: d8b4adaa1ac036e66ee6057895dc657c2bb66368

File: content/en/docs/demo/services/cart.md [hunk 3]
```diff
@@ -91,4 +92,11 @@ builder.Services.AddOpenTelemetry()
 
 ## Logs
 
-TBD
+Logs should be configured in .NET dependency injection container on
```
Comment (mviitane) [inline content/en/docs/demo/services/cart.md:95]:
```suggestion
Logs are configured in the .NET dependency injection container on
```

**Review by mviitane — APPROVED** [review]
Looks great! Added only one minor comment.

**Pull request merged**